wp-content/themes/core/theme.json

```diff
@@ -1,10 +1,77 @@
 {
-	"$schema": "https://schemas.wp.org/trunk/theme.json",
-	"version": 2,
-	"settings": {
-		"layout": {
-			"contentSize": "80ch",
-			"wideSize": "1200px"
-		}
-	}
+  "$schema": "https://schemas.wp.org/trunk/theme.json",
+  "version": 2,
+  "settings": {
+    "color": {
```

Comment:
❓ I left background color settings enabled. We disabled it for LendNation. Do we want it disabled here to start?

Reply:
I'd say yes for now.